integrations/hitl-salesforce/src/events/conversation-message.ts

```
@@ -93,6 +93,17 @@ export const executeOnConversationMessage = async ({
         }
       }
       break
+    case 'RichLink':
+      const richLinkContent = entryPayload.abstractMessage.staticContent as RichLinkStaticContent
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)